curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/65ca2d0625aa3a3201067f70.md

```diff
@@ -0,0 +1,150 @@
+---
+id: 65ca2d0625aa3a3201067f70
+title: Step 30
+challengeType: 0
+dashedName: step-30
+---
+
+# --description--
+
+In the callback function of your `forEach`, use the `el` parameter to access the `counts` object. If the element is not in `counts`, set it to `0`. Otherwise, increment the count for the element. 
+
+Below your `forEach`, log out the final result of `counts`.
```

Comment:
The directions are missing the instructions to test it out with 4,4,2,5.  It is also missing the instructions for returning counts. Right now the start code returns array which will cause the test to fail. The instructions and tests should be update to accommodate that. 

````suggestion
In the callback function of your `forEach`, use the `el` parameter to access the `counts` object. If the element is not in `counts`, set it to `0`. Otherwise, increment the count for the element. 

Below your `forEach`, log out the final result of `counts`.

Then change your return statement to return `counts` instead of `array`.

To test this, enter the numbers `4, 4, 2, 5` and click `Calculate`. You should see the following in the console:

```plaintext
{ '2': 1, '4': 2, '5': 1 }
```
````

Reply:
> 65ca2d0625aa3a3201067f70.md: 18: prism-langs Code block languages should be supported by PrismJS ['plaintext' is not recognised.]

I got this error message for the line 18